src/server/bootstrap.rs

```diff
@@ -56,22 +56,29 @@ pub async fn init_hq_server(
 }
 
 pub async fn get_client_connection(server_directory: &Path) -> anyhow::Result<ClientConnection> {
+    let mut default_home = dirs::home_dir().unwrap_or_else(std::env::temp_dir);
```

Comment:
Use `default_server_directory_path` to avoid unexpected changes in the default directory. Move this function to `client/mod.rs` to make it available here.